<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtnpNewFileDialog</class>
 <widget class="QDialog" name="QtnpNewFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>99</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="TemplateLayout">
       <item>
        <widget class="QLabel" name="TemplateLabel">
         <property name="text">
          <string>Template:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="TemplateCB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="widthLabel">
         <property name="text">
          <string>Width:</string>
         </property>
         <property name="buddy">
          <cstring>widthSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="widthSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="heightLabel">
         <property name="text">
          <string>Height:</string>
         </property>
         <property name="buddy">
          <cstring>heightSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="heightSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>768</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="GridAndCoordWidgetLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QtnpNewFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
